### ISSUE TITLE
transloco:migrate says "Nothing to be done."

### ISSUE THREAD

**Issue by ygeelen-tb** [ngneat/transloco#203]
Hello!

This library looks very promising. We're using @angular/core 6.0.0 and I installed @ngneat/transloco 2.12.1 following the manual at https://netbasal.gitbook.io/transloco/installation

Also I'd like to migrate all my Angular i18n translations to transloco.
The command runs but doesn't do any changes to any files and says "Nothing to be done." ?
(Also I needed to create 1 dummy spec file to not get any error.)

npm run ng g @ngneat/transloco:migrate --path "/src/app" --from "Angular"

> trusto-box@0.0.0 ng C:\projects\TrustO\DigitallyRegisteredMail\sources\DRM.Web
> ng "g" "@ngneat/transloco:migrate" "/src/app" "Angular"

Starting migration script

Migrating HTML files �📜
√ Step 1/2: Migrating directives
√ Step 2/2: Migrating pipes

Migrating TS files �📜
√ Step 1/5: Migrating modules
√ Step 2/5: Migrating service imports
√ Step 3/5: Migrating constructor injections
√ Step 4/5: Migrating service usage
√ Step 5/5: Migrating specs

              �🌵 Done!🌵🌵
Welcome to a better translation experience �🌐

For more information about this script please visit �👉 https://github.com/ngneat/transloco/tree/v1/migration/migration.md

Nothing to be done.

**Comment by shaharkazaz** [comment]
@ygeelen-tb 
> (Also I needed to create 1 dummy spec file to not get any error.)
What do you mean by that? what error did you get?

Semes like according to the log that the incorrect migration ran, since Angular's i18n doesn't need to iterate `ts` files.
Can you please try running `ng g @ngneat/transloco:migrate` and follow the cli inputs?

**Comment by ygeelen-tb** [comment]
I simply got the error that Step 5/5: Migrating specs didn't succeed.

When following the CLI inputs it worked (though I had to install the npm package fs-extra first) !
So the migrate from Angular i18n didn't replace the i18n calls from TypeScript?

**Comment by shaharkazaz** [comment]
@ygeelen-tb What would you expect to have been done in the TS files? which calls?

**Comment by ygeelen-tb** [comment]
Calls like this in components could be replace with TranslocoService calls:

import { I18n } from '@ngx-translate/i18n-polyfill';

constructor(private i18n: I18n) {
    this.totalCountText = i18n("Totaal");
}

**Comment by shaharkazaz** [comment]
@ygeelen-tb This is a 3rd party library and part of the official Angular i18n and therefore not part of this migration script.
You can open a PR if you want to add support for it, but currently, we don't have plans to add support.

The schematics issue will be fixed.

**Comment by ygeelen-tb** [comment]
@shaharkazaz Thanks!

By the way, for my initial installation of transloco to work with TypeScript, Angular 6 and rx.js 6, I had to upgrade multiple packages:

typescript: ~2.7.2 to ~3.1.1
@angular-devkit/build-angular: ~0.6.1 to ~0.10.0
@angular/cli: ~6.0.1 to ~7.0.2
@angular/compiler-cli: ^6.0.0 to ^7.0.0
@angular/language-service: ^6.0.0 to ^7.0.0

Especially TypeScript, because the typescript definition file of the transloco loader could not be understood by TypeScript 2.7.2. Because of the TypeScript upgrade, I needed to upgrade the other packages listed above.
